_data/vendors.yml

```diff
@@ -417,6 +417,15 @@
   pricing_source: https://www.twilio.com/enterprise
   updated_at: 2018-10-22
 
+- name: Veremark
+  url: https://www.veremark.com
+  base_pricing: pay as you go
+  sso_pricing: £100 per month
+  percent_increase: 100% [^veremark]
```

Comment:
Technically, this is 9900% increase in pricing, if you're going by 1GBP -> 100GBP.  (100% increase would be 2GBP)

Reply:
It's a fair point, though I feel like the 100% understates how significant the tax is. perhaps the right answer here is to use the infinite symbol instead?

Reply:
∞ works too, i guess.

Reply:
i added ∞ now lol